openml/config.py

```diff
@@ -351,27 +362,32 @@ def get_cache_directory():
     """
     url_suffix = urlparse(server).netloc
     reversed_url_suffix = os.sep.join(url_suffix.split(".")[::-1])
-    _cachedir = os.path.join(cache_directory, reversed_url_suffix)
+    _cachedir = os.path.join(_cache_directory, reversed_url_suffix)
     return _cachedir
 
 
-def set_cache_directory(cachedir):
-    """Set module-wide cache directory.
+def set_root_cache_directory(root_cache_directory):
+    """Set module-wide base cache directory.
 
-    Sets the cache directory into which to download datasets, tasks etc.
+    Sets the base cache directory that defines how the actual cache
+    directory for the server being used is derived. This is
+    ``root_cache_directory / top-level domain / second-level domain /
+    hostname``, and by default is set to
+    ``root_cache_directory / org / openml / www`` for the standard
+    OpenML.org server.
```

Comment:
```suggestion
    Sets the root cache directory wherein the cache directories are created to store content
    from different openml servers. For example, by default, cached data for the standard openml.org
    server is stored in ``root_cache_directory / org / openml / www``.
```
I found the description a bit vague, due to inconsistent use of base vs. root and that it does not really define how the cache directory is derived, rather which directory the caches are placed in. This is just a suggestion, but any update the removes this ambiguity is fine with me.

Reply:
Changed this, thanks for spotting that I forgot to replace `base` here with `root`.